features/specs/S-en-T-test-omgekeerd-ouders/Tg016.feature

```diff
@@ -599,10 +599,10 @@ Functionaliteit: Tg016 - Petersen-Perquin & Peereboom-Peek & Pauwels-Platje
       | naam                | waarde    |
       | burgerservicenummer | 000000048 |
     En heeft de persoon een 'gezag' met de volgende gegevens
-      | naam                             | waarde    |
-      | type                             | Voogdij   |
-      | minderjarige.burgerservicenummer | 000000048 |
-    En heeft 'gezag' geen derden
+      | naam                             | waarde                                                  |
```

Comment:
dit resultaat begrijp ik niet.
- minderjarige heeft 1 ouder en gezag uitspraak 1D.
- ouder heeft partner die was overleden voor minderjarige geboren is, dus minderjarige is niet in huwelijk geboren
- ouder is overleden, dus heeft alleen de derde het gezag
- dan verwacht ik voogdij voor onbekende derde
- de toelichting "beide ouders overleden" kan ook niet, want minderjarige heeft maar 1 ouder.
-

Reply:
Er was verwarring ontstaan wat wij verstaan onder niet ouder (derde) vanuit het domein en niet ouder vanuit gezag-api. gezag-api ging er altijd van uit dat niet-ouder gelijk staat aan de partner van de desbetreffende ouder.

Dat is rechtgetrokken en deze uitspraak resulteert weer in voogdij zonder derden.